### PR TITLE
Core Role Adjustment Beta V2 (CRAB2)

### PR DESCRIPTION
**Pull request by Celyne-git**
The Core Role Adjustment Beta changes the job slot amounts of many core roles (and also allows for matrixing of dead bodies, incidentally).

**NCR**
1 Captain
1 Ambassador
1 Lieutenant
2 Sergeant
2 Specialist
4 Trooper
1 Veteran Ranger
2 Ranger
4 NCR Assistant

**Legion**
1 Centurion
1 Orator
2 Decanus
2 Vexillarius
2 Explorer
4 (Veteran) Legionary
3 Auxilia
4 Camp Follower

As stated in the title, this is a beta- these changes may not be permanent, and will be reviewed after a sufficient time has passed.

BOS Initiate Knight/Scribe: now whitelisted
Raiders and Great Khans have also had their join text updated to be more clear about their roles.
Also, 10mm SMGs are bulky, 30-round 556 magazines are rarer, and the Specialist starts with a pistol instead of a rifle.

### CODE REVIEW

**Comment by Kabramen** [comment]
>Raider: 0 slots, down from 10 - now manually spawned in by staff upon approval of gimmick

What's the point of have a raider whitelist, when you can't play the role when there's no admins on? Come on, instead of inprove raiders  to have less shitters, you're making the role not only less fun to play but aso way less accessible. well, unless is your plan of just remove raider by entirely.

If anything, most of raiders player with go play as khans ( which **is** a rader gang, with still no whitelisting for some reason) and move the problems of a role to another.

I'm sorry but thats a terrible change, and feels like you're out of touch with the community. such controversy chage should be announced and discuss more carefully before doing the PR. Hell, maybe even having a poll.

For the rest of PR, I have nothing to complain

**Comment by Celyne-git** [comment]
Edited the PR with the following:

**How the raider thing works:** Rather than spawning in as a Raider and then going around doing Raidery things, you'll instead adminhelp and go "hey, I'd like to be a raider, here's my idea of how to make the round interesting". If your idea is good, you'll then be given the tools necessary to make that happen- if you want to be the Mechanist with a pet cyborg, we'll give you a pet cyborg, or if you want to be a corrupt Deputy, we'll give you a Deputy outfit. You get more freedom to pursue interesting and unique ideas that aren't just 'give caps or die', and you're provided some things to make that happen.

The whitelist is there so we know who's been approved for shenanigans. If you're not whitelisted, you're not likely to get spawned in as a raider.

As for Khans, they now receive join text stating that they aren't raiders, and shouldn't act as one without staff approval.

**Comment by Manfred21** [comment]
I disagree fundamentally with the raider change in every way. I've said my piece on discord, though.

**Comment by Kabramen** [comment]
>As for Khans, they now receive join text stating that they aren't raiders, and shouldn't act as one without staff approval

you should change to "they **aren't generic** raiders, and should follow they faction lore" in that text, because saying they aren't is totally lore breaking

image

image

Organized or not, they're a still raiders, and should recive the same tratament in the whitelist. That text itself won't stop people acting silly, or even change the fact that they are raiders.

A better soltion for raiders would be to add in-games gangs (like jackals and vipers), in which
 has a established lore and making easy as raider to know what they can or can not do.

Finally, people already ahelped before to make special raider shenanigans, removing raiders to force it is pointless and annoying.

**Comment by Celyne-git** [comment]
Raiders made regular-spawnable again per feedback, and have been given a nifty new text when they join to clarify their role and encourage them to be interesting.

**Comment by captainwiseguy** [comment]
getting mugged in the wasteland should be very common if you arent prepared, discouraging that will only lead to several gimmicks being done, until those gimmicks become old and repeated, and there are no fresh AND exciting gimmicks to be done anymore

**Comment by ProfessorPopoff** [comment]
Preposterous. absolutely abhorrent. Never in my life have I ever seen a more disgusting pull request. Good job good chap. 

fucks wrong with you

**Comment by Hekzder** [comment]
My disappointment is immeasurable and my day is ruined.

**Comment by Zebraman** [comment]
Legionary: 2 slots, down from 12 - given tougher gear (shields, veteran armor)
NCR Trooper: 2 slots, down from 12

I don't get it. What's the point of having command roles if they have no one to command? And the Legion is still cucked by having less combat role. This whole PR make no sense and doesn't improve anything, it just make faction more of a joke than what they currently are.

**Comment by Dr-Oreo** [comment]
Okay, many problems with it. Major one being:

You have lowered the numbers of troops of each faction, for some reason, down to two. This is absolutely silly. I get lowering it down from high numbers if you want less packed factions, but this is far too much. Having eight recruits, six legionnaires, two explorers and two decanii would be far better. It gives more for command roles to do and forces less players to sit out the round or play something like waster, which seems to be what the server is pushing toward.

This is only going to create more incompetent in command roles, as the players who want to play a faction, but aren't that aware of their lore or how to command, will be forced to spawn as the only roles available. And once the TWO slots of each non command roles are taken, they'll be forced into command.

A lot of players don't want to play wasters or town, instead they want to interact as a faction and use that lore to play in the game world. Factions now, however, are just a pushover. A single powergame geared person could easily wipe out a faction due to these lowered numbers.

You've also made it so AUXILIA and CAMP FOLLOWERS outnumber Legion troops, and I shouldn't have to explain why that's stupid.

TL;DR

You've turned factions weaker which will only worsen gameplay, I won't be surprised if you start getting little to no faction players from this point on if the change stays. By lowering numbers, you've removed what makes factions fun and have made them no different from a raider gang with potentially weaker gear since they have to wear uniform.

Downright awful change.

**Comment by NoctuaAngelis** [comment]
Hey, I looked at this and after doing some thinking maybe a better solution. I ran it by Marrone and he seemed to like this  as well.
I prefer leaving Legionairres at 6-8 but that's just me.. here
-------
BOS Initiate Knight/Scribe: now whitelisted
NCR Lieutenant: 2 slots, down from 3
NCR Sergeant: 2 slots, down from 3
NCR Specialist: 2 slots, down from 3
NCR Trooper: 4 slots, down from 12
NCR Ranger: 3 slots, down from 4
NCR Recruit changed to NCR Assistant: 4 slots, noncombat role. Unwhitelisted.
Legion Decanus: 2 slots, down from 3
Legion Explorer: 2 slots, down from 3
Veteran Legionary: 2 slots, up from 0 - given tougher gear (shields, veteran armor)
Legionary: 2 slots, down from 12 
Legion Auxilia: 3 slots, down from 5
Legion Recruit: 0 slots, down from infinite
Camp Follower: 5 slots, down from infinite

**Comment by Zebraman** [comment]
> ## Hey, I looked at this and after doing some thinking maybe a better solution. I ran it by Marrone and he seemed to like this as well.
> I prefer leaving Legionairres at 6-8 but that's just me.. here
> BOS Initiate Knight/Scribe: now whitelisted
> NCR Lieutenant: 2 slots, down from 3
> NCR Sergeant: 2 slots, down from 3
> NCR Specialist: 2 slots, down from 3
> NCR Trooper: 4 slots, down from 12
> NCR Ranger: 3 slots, down from 4
> NCR Recruit changed to NCR Assistant: 4 slots, noncombat role. Unwhitelisted.
> Legion Decanus: 2 slots, down from 3
> Legion Explorer: 2 slots, down from 3
> Veteran Legionary: 2 slots, up from 0 - given tougher gear (shields, veteran armor)
> Legionary: 2 slots, down from 12
> Legion Auxilia: 3 slots, down from 5
> Legion Recruit: 0 slots, down from infinite
> Camp Follower: 5 slots, down from infinite

Raises the number of regular legionary to 4 to match the number of trooper the NCR get, so the vet legionary would be the counterpart of the ranger and it would even the number of fighting role. 6/8 basic grunt in each faction would also be better. Also, are vex still a thing with this PR? They are not mentioned anywhere.

**Comment by Dr-Oreo** [comment]
Yeah, new changes are still no good. I don't understand, why are you trying to limit factions? Because of TDM? Right now it seems like you're just making them more useless than giving them a different gameplay loop.

One: Too many damn Auxilia and Followers. You do not need that many. You only need two followers to get everything you need in one game, having three slots are enough.

Two: Numbers are still so goddamn low that it's no better than a raider gang.

Three: NCR Assistant seems like another useless role. If you're trying to make Sharecroppers, make them a separate faction that NCR protects outside of their base. Having NCR farmers in their base defeats the advantage Legion has, which is the ability to easily produce resources. This works from a gameplay perspective and a lore perspective. Lore wise, NCR are struggling to maintain supplies, they're unable to produce their own and must rely on weak supply lines or the supplies of local towns. Having it so NCR don't have easily producible supplies SHOULD force them to interact with town's farmers. It at least used to before everyone learnt how to farm. Since it may be too much work to balance farming properly, good solution. just make sharecroppers and put them in the unused slums below NCR which has many farms. Remove the farm from NCR base.

Anyway, here are better numbers. 

NCR
1 Captain
1 Ambassador
2 Lieutenant
2 Sergeant
4 Specialist
6 Trooper
1 Veteran Ranger
2 Ranger
2 Sharecropper

Legion
1 Centurion
1 Orator
2 Decanus
3 Veteran Legionnaires
6 Prime Legionary
1 Vexillarius
2 Explorer
2 Auxilia
2 Camp Follower

A decent amount of numbers, lowered far from the original yet still enough to make it fun to play. If you want to make it fun, nerf and buff different playstyles. Lower the amount of DECENT weaponry on either faction's side. Buffing them only encourages TDM, remove marksman carbines from sergeant's. Veteran Legion spawn with lever action shotgun and a fireaxe. Sergeant spawns with hunting shotgun and wooden baton. The only people who should be getting powerful weaponry are Vet Ranger, Centurion and Captain, with Centurion having equivalent close range power to a Vet ranger.

REALLY FOCUS on making it so weapons are fair, it encourages TDM when you have uber powerful weapons and want to use them on people. If people have shittier ones, well, they'll want to get into fights less. Make it so >Legion KINGS of close range >NCR KINGS of long range

Now. I get what you're trying to do with all this, you want to make factions less TDM-focused. That's not going to change by hugely lowering numbers, it's just going to mean they get absolutely creamed when they do go to TDM. Instead, focus on giving players the incentive to go out and roleplay in a more peaceful style. Have rewards via completing round by round objectives.

**Comment by CaputCapital** [comment]
The new changes despite being a beta are very flawed. NCR assistants are just recruits without a vest. They can fight compared to Legion camp followers. I recall one round where NCR had more than twice the numbers than Legion. Take a look at this:
Bad Ratio

**Comment by Celyne-git** [comment]
The imbalance is an issue with the players, not the code- that just shows people want to play NCR more than they want to play Legion. NCR Assistants are still noncombatants, like camp followers, and will still be slapped if they're actively taking part in offensive operations.

**Comment by CaputCapital** [comment]
So players want to play NCR more than they want to play Legion. Why? One reason from what I've gathered is that it's been for the weapons NCR gets. Such as assault carbines and the Anti Material Sniper Rifle, exclusive weapons Legion does not receive for any role.
Comparison
